src/index.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import ReactDOMClient from 'react-dom/client'
 import App from './app.jsx'
 import { RouterProvider, type Route } from './context/router-context.jsx'
+import * as renderChecks from './lib/routing-render-checks.js'
```

Comment:
i'll push a change up updating this for the rest of the imports.. it would be cleaner in this file.

Reply:
pushed up [`b011e46` (#293)](https://github.com/ipfs/service-worker-gateway/pull/293/commits/b011e46ac4bffa68ac972d177b4cc40af309ff1a)

Reply:
Nice! I totally missed that.